src/ui/menu/Export.qml

```diff
@@ -91,35 +74,41 @@ MenuItem {
         }
     }
 
-    function applyOutputSize() {
-        controller.set_output_size(outWidth, outHeight);
-    }
-    function setOutputSize(w, h) {
-        orgWidth = w;
-        orgHeight = h;
-        outputHeight.preventChange2 = true;
-        outputWidth.preventChange2 = true;
-        outputWidth.value = w;
-        outputHeight.value = h;
-        outputWidth.preventChange2 = false;
-        outputHeight.preventChange2 = false;
-    }
-    function updateOutputSize(isWidth) {
-        if (lockAspectRatio.checked && ratioHeight > 0) {
-            const ratio = ratioWidth / ratioHeight;
-            if (isWidth) {
-                outputHeight.preventChange2 = true;
-                outputHeight.value = Math.round(outputWidth.value / ratio);
-                outputHeight.preventChange2 = false;
+    property bool disableUpdate: false;        
+    function notifySizeChanged() {
+        Qt.callLater(() => controller.set_output_size(outWidth, outHeight));

```

Comment:
This should be `Qt.callLater(notifySizeChanged)` instead on the caller side.
`callLater` will debounce multiple calls to the same function and will only execute it once. But only for named functions, so lambdas will be called many times